content/people/david-lebauer/index.md

```diff
@@ -27,4 +27,11 @@ caption = ""
 
 +++
 
-David steers this ship
\ No newline at end of file
+
+Bioscience Research Labs Room 207
```

Comment:
This is redundant with what's in the contact section on the main page? If this contact info is included on David's, everyone's page should have it.  

It's also not formatted correctly: 
![Screen Shot 2020-06-26 at 11 32 16 AM](https://user-images.githubusercontent.com/6686508/85890542-6aa4ed80-b7a2-11ea-803a-f08f00fd57fb.png)